packages/beacon-node/src/network/gossip/validation/queue.ts

```diff
@@ -17,6 +17,8 @@ const gossipQueueOpts: {[K in GossipType]: Pick<JobQueueOpts, "maxLength" | "typ
   [GossipType.attester_slashing]: {maxLength: 4096, type: QueueType.FIFO},
   [GossipType.sync_committee_contribution_and_proof]: {maxLength: 4096, type: QueueType.LIFO, maxConcurrency: 16},
   [GossipType.sync_committee]: {maxLength: 4096, type: QueueType.LIFO, maxConcurrency: 64},
+  [GossipType.light_client_finality_update]: {maxLength: 1024, type: QueueType.FIFO},
+  [GossipType.light_client_optimistic_update]: {maxLength: 1024, type: QueueType.FIFO},
```

Comment:
What the reasoning for 1024? This numbers should be researched to be a max of what's necessary for regular operations and limiting memory as much as possible